### PR TITLE
AST coverage tests

### PR DESCRIPTION
**Pull request by jgberry**
Added comprehensive coverage tests for `ssort._ast.iter_child_nodes`.

### CODE REVIEW

**Comment by jgberry** [comment]
Unfortunately this doesn't completely cover all of the `_ast` module because there are some edge cases which are not reached using this test generation methodology.

This should, however, satisfy #34 as any AST nodes introduced in future versions will cause a test failure.

File: src/ssort/_ast.py [hunk 1]
```diff
@@ -365,6 +365,60 @@ def _iter_child_nodes_of_index(node: ast.Index) -> Iterable[ast.AST]:
         yield node.value
 
 
+@iter_child_nodes.register(ast.Load)
+@iter_child_nodes.register(ast.Store)
+@iter_child_nodes.register(ast.Del)
+def _iter_child_nodes_of_expr_context(
+    node: ast.expr_context,
+) -> Iterable[ast.AST]:
+    return ()
```
Comment (bwhmather) [inline src/ssort/_ast.py:374]: Perhaps we should explicitly skip these in the tests.  I think they are really attributes of the parent node.

Comment (jgberry) [reply src/ssort/_ast.py]: Yes, I think you're right. In fact, I think all the operators can be skipped as well.

**Comment by jgberry** [comment]
Ok, turns out I wasn't iterating over the generator return by `iter_child_nodes` which explains the previous lack of increase in coverage. Now we are getting a respectable 1.2% bump.

**Comment by bwhmather** [comment]
Tests for the test throne!

**Review by bwhmather — APPROVED** [review]
